File: Wirtualny świat/java/src/organizm/wilk.java

package organizm;
import swiat.swiat;


public class wilk extends zwierze {
    public wilk(swiat WielkiSwiat, int x, int y)
    {
        super(WielkiSwiat, x, y);
        sila = 9;
	inicjatywa = 5;
        this.nazwa = 'W';
    }
   
    @Override
    public void nowy(swiat WielkiSwiat, int x, int y)
    {
        organizm o = new wilk(WielkiSwiat, x, y);
    }
    public String nazwa()
    {
        return "Wilk";
    }
    
}

File: Wirtualny świat/C++/ConsoleApplication18/wilk.cpp
#include "stdafx.h"
#include "wilk.h"


wilk::wilk(swiat& ref, int x, int y):zwierze(ref,x,y)
{
	this->sila = 9;
	this->inicjatywa = 5;
	this->nazwa = 'W';
	this->specjalna_kolizja = 0;
}

char wilk::rysowanie() 
{
	return this->nazwa;
}
void wilk::nowe(swiat& ref, int x, int y)
{
	organizm *o = new wilk(ref, x, y);
	cout << "dodano: " << this->rysowanie() << " na polu: " << x + 1 << ':' << y + 1 << ":\n";
}
wilk::~wilk()
{
}
